matcher/solvers/minmax_solver.py

```diff
@@ -42,12 +44,32 @@ def __init__(
         self.flow_matrix = None
         self.optimal_cost = None
         self.cost = None
-
         self.logger = logger
 
+    def _validate_input_range(self):
```

Comment:
Ohhh I see, we need to do this validation in MinMaxSolver not in the SimpleSolver, cool

Reply:
is there a test to check that this is happening?

Reply:
Yeah, setting strict to True while running simpleSolver for minimums was not the solution. MinMaxSolver itself needs to validate this.

Reply:
I will add a test for this.

Reply:
Test added.